Resources/Storage/DIPStorage.py

```diff
@@ -500,3 +500,16 @@ def getDirectory( self, path, localPath = False ):
     resDict = {'Failed':failed, 'Successful':successful}
     return S_OK( resDict )
 
+  def getOccupancy(self):
+    """ Gets the DIPStorage occupancy info
+    """
+    rpc = RPCClient(self.getURLBase(), timeout=120)
+    free = rpc.getFreeDiskSpace("/")
```

Comment:
What is the parameter ? 
Shouldn't the StorageElementHandler answer with the space free on the partition where it stores its data ?

Reply:
It's the path, basically it will export the result of `st = os.statvfs(path)`

Reply:
Then I would say it is incorrect. Take LogSE as an example: it has a special partition

Reply:
Yes I see, and I agree with you. I just discussed it with Andrei, he will implement this properly.